net-irc/kvirc/kvirc-5.2_pre20210630233331.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 2009-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
```

Comment:
@Arfrever We have already agreed for net-misc/cadaver not to force downgrade on our users, yet you have ignored it for this package and did against what we discussed.